src/Components/v2/__stories__/CarouselV3.story.tsx

```diff
@@ -44,6 +45,7 @@ storiesOf("Styleguide/Components/CarouselV3", module)
                 src={props.resized.url}
                 width={props.resized.width}
                 height={props.resized.height}
+                key={props.resized.url}
```

Comment:
Did adding this key fix the flash?

Reply:
:( nope

Reply:
Unless it's needed (errors in the console) then we can omit keys